internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -2626,11 +2626,10 @@ components:
     FilesystemTyped:
       type: object
       required:
-        - mountpoint
```

Comment:
Should there be a test for mountpoint added to the non-swap types in the code since it's now possible to pass a non-swap type without setting the mountpoint.

Reply:
This case (not the test) is covered by the `bpDisk.Validate()` call when converting the Disk Customizations from the request to BP customization.

So a test case should be added :+1:

Reply:
Amended the last commit to add more tests for the mountpoint vs swap cases.